src/main/java/org/opensearch/neuralsearch/plugin/NeuralSearch.java

```diff
@@ -52,15 +50,9 @@ public class NeuralSearch extends Plugin implements ActionPlugin, SearchPlugin,
      * Gates the functionality of hybrid search
      * Currently query phase searcher added with hybrid search will conflict with concurrent search in core.
      * Once that problem is resolved this feature flag can be removed.
-     * Key is the name string, value is key + transport feature specific prefix,
-     * prefix is added by core when we register feature (https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/plugins/PluginsService.java#L277)
-     * We need to write and read by the value, key is only for definition
      */
     @VisibleForTesting
-    public static final Pair<String, String> NEURAL_SEARCH_HYBRID_SEARCH_DISABLED = Pair.of(
-        "neural_search_hybrid_search_disabled",
-        String.join(".", TransportSettings.FEATURE_PREFIX, "neural_search_hybrid_search_disabled")
-    );
+    public static final String NEURAL_SEARCH_HYBRID_SEARCH_ENABLED = "neural_search_hybrid_search_enabled";
```

Comment:
The variable is already public we can remove @VisbleForTesting

Reply:
I made it public only because it's required for tests otherwise private will be enough, my understanding is that @VisbleForTesting is required in this case to mark that it's not by mistake.

Reply:
oh.. sorry my bad. I read it wrong then